docs/usage.md

```diff
@@ -769,6 +777,32 @@ The chromosome in your FastA reference that you'd like to be treated as circular
 
 If you want to filter out reads that don't map to a circular chromosome, turn this on. By default this option is turned off.
 
+#### Bowtie2
+
+##### `--bt2_alignmode`
+
+The type of read alignment to use. Options are 'local' or 'end-to-end'. Local allows only partial alignment of read, with ends of reads maybe 'soft-clipped', if provides best alignment score. End-to-end requires all nucleotides to align somehow.  Default is 'local', following [Cahill et al (2018)](https://doi.org/10.1093/molbev/msy018) and [(Poullet and Orlando 2020)](https://doi.org/10.3389/fevo.2020.00105).
+
+##### `--bt2_sensitivity`
+
+The Bowtie2 'preset' to use. Options: 'no-preset' 'very-fast', 'fast', 'sensitive', or 'very-sensitive'. These strings apply to both `--bt2_alignmode` options. See the Bowtie2 [manual](http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#command-line) for actual settings. Default is 'sensitive' (following [Poullet and Orlando (2020)](https://doi.org/10.3389/fevo.2020.00105), when running damaged-data _without_ UDG treatment)
+
+##### `--bt2n`
+
+The number of mismatches allowed in the seed during seed-and-extend procedure of Bowtie2. This will override any values set with `--bt2_sensitivity`. Default: 0.
```

Comment:
```suggestion
The number of mismatches allowed in the seed during seed-and-extend procedure of Bowtie2. This will override any values set with `--bt2_sensitivity`.  Accepted values: 0 or 1. Default: 0.
```